packages/server/src/routes/grants.js

```diff
@@ -348,7 +348,12 @@ router.put('/:grantId/assign/agencies', requireUser, async (req, res) => {
     }
 
     await db.assignGrantsToAgencies({ grantId, agencyIds, userId: user.id });
-    email.sendGrantAssignedEmail({ grantId, agencyIds, userId: user.id });
+    try {
+        email.sendGrantAssignedEmail({ grantId, agencyIds, userId: user.id });
```

Comment:
Should this be `await email.sendGrantAssignedEmail` to ensure that it waits for the function to complete to catch the error?

Reply:
Yes, good catch! And that probably also explains why the second test I was trying to write wasn't working... let me see if I can get a test for the 500 response as well

Reply:
Ugh, I think the test is unfortunately not worth the effort (unless you have a quick read on how to get it working). I was trying to add something like this to the `PUT /api/grants/:grantId/assign/agencies` tests in `__tests__/api/grants.test.js`: 

```node
            it('gracefully returns 500 response for unexpected email errors', async () => {
                const emailSpy = sandbox.spy(email, 'sendGrantAssignedNotficationForAgency');
                sandbox.stub(email, 'buildGrantDetail').throws();
                const response = await fetchApi(`/grants/${assignEndpoint}`, agencies.own, {
                    ...fetchOptions.admin,
                    method: 'put',
                    body: JSON.stringify({ agencyIds: [agencies.own] }),
                });
                expect(response.statusText).to.equal('Internal Server Error');
                expect(emailSpy.notCalled).to.equal(true);
            });
```

I believe the issue is that our existing tools for stubbing and throwing don't have access to stub out the method that's getting called, since it's really more of an integration test than a unit test here. (Otherwise, we could use `rewire` or `sinon` to mock out the function the code is calling.) Anyway, I'm going to move on, but if you have the answer here, I'd love to hear it!